examples/jsm/renderers/common/Renderer.js

```diff
@@ -773,6 +775,16 @@ class Renderer {
 		let renderTargetData = null;
 		const renderTarget = this._renderTarget;
 
+		if ( renderTarget === null ) {
+
+			stencil = this.autoClearStencil;
+
+		} else if ( renderTarget.stencilBuffer === false ) {
+
+			stencil = false;
+
+		}
```

Comment:
If I'm understanding right, as this is written you won't be able to clear a render target with a stencil buffer enabled if stencil is disabled on the renderer, since stencil is set to false at the beginning of this function.